nixos/modules/system/boot/loader/systemd-boot/systemd-boot.nix

```
@@ -29,6 +29,22 @@ let
     inherit (efi) efiSysMountPoint canTouchEfiVariables;
 
     memtest86 = if cfg.memtest86.enable then pkgs.memtest86-efi else "";
+
+    netbootxyz = if cfg.netbootxyz.enable then pkgs.netbootxyz-efi else "";
+
+    copyExtraFiles = pkgs.writeShellScript "copy-extra-files" ''
+      empty_file=$(mktemp)
+
+      ${concatStrings (mapAttrsToList (n: v: ''
+        ${pkgs.coreutils}/bin/install -Dp "${v}" "${efi.efiSysMountPoint}/${n}"
```

unlikely to cause an error in practice, but one more nit:
```suggestion
        ${pkgs.coreutils}/bin/install -Dp "${v}" "${efi.efiSysMountPoint}"/${escapeShellArg n}
```
also for the others.

---

Fixed :+1: